pkgs/development/compilers/ponyc/default.nix

```diff
@@ -1,85 +1,115 @@
-{ stdenv, fetchFromGitHub, llvm, makeWrapper, pcre2, coreutils, which, libressl, libxml2,
-  cc ? stdenv.cc, lto ? !stdenv.isDarwin }:
+{ clangStdenv, fetchFromGitHub, fetchurl, llvm, makeWrapper, pcre2, coreutils, which, libressl, libxml2, cmake, z3, substituteAll,
```

Comment:
I think it still makes sense to accept the `stdenv` as an argument. You can override `stdenv = clangStdenv;` in the pkgs/top-level/all-packages.nix.

Reply:
I can do that, I just need to ensure that I leave a comment in both locations with the appropriate reference.